app/features/recent-list/reducer.js

```diff
@@ -65,6 +72,21 @@ function _insertConference(
     return newRecentList;
 }
 
+/**
+ * Remove a conference from the recent list array.
+ *
+ * @param {Array<RecentListItem>} recentList - Previous recent list array.
+ * @param {RecentListItem} toRemove - Conference to be removed.
+ * @returns {Array<RecentListItem>} - Updated recent list array.
+ */
+function _removeConference(
+        recentList: Array<RecentListItem>,
+        toRemove: RecentListItem
+): Array<RecentListItem> {
+    return recentList.filter(
+        (conference: RecentListItem) => conference.room !== toRemove.room);
```

Comment:
You also need to check the serverURL field, or you'd end up removing all conferences with the same name, no matter the server.

But actually, why not remove all elements which are not `toRemove` ?

Reply:
Good point, not sure why I added the `.room`s